query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseObjectField.java

```diff
@@ -100,7 +101,7 @@ public void setPath(List<Object> path) {
         .addAll(
             getFields()
                 .stream()
-                .map(field -> field.getErrorCodes())
+                .map((Function<Field, Set>) Field::getErrorCodes)
```

Comment:
Why do we need to type cast this if the `Field` interface defines `getErrorCodes` as returning `Set<String>`?